src/lib/tutorial.js

```diff
@@ -9,7 +9,9 @@ export function isTutorial() {
 export function display(t) {
   const isSmall = document.querySelectorAll('.coz-nav')[0].offsetParent === null
   const isLandscape = window.innerWidth > window.innerHeight
-  const cozyBarMenuClass = isSmall ? '.coz-bar-burger' : '[data-icon=icon-cube]'
+  const cozyBarMenuClass = isSmall
+    ? '.coz-bar-burger'
```

Comment:
Wow I missed that the first time. We should not rely on a selector with graphic information. Is it possible to avoid it (I assume that it is not if you did not do it).

Reply:
The thing is that I think we are reproducing the same mistake which caused the current regression, using strongly contextualized class name like `.coz-bar-burger`. The role of this class name is to provide a burger icon. It should not be used as a reference for a JS implemented behaviour. I suggest to add a dedicated class to the button, like `class="apps-button"` or even other attributes like `id="apps-button"` or `data-tutorial="apps"`. I know that it's a little bit more expensive as we need to update the bar, but I am afraid that we'll be facing same regression soon if we not switch to a clever way. What do you think ?

Reply:
I think you're right. I like `data-tutorial="apps"` better because it highlights the purpose, whereas I'm afraid a generic `id` might get deleted some day because it wouldn't be obvious why it's there.

Reply:
Yes, I just closed it so the PR doesn't attract attention while it's being worked on (I guess that didn't work ¯\\\_(ツ)_/¯)